spec/controllers/spree/credit_cards_controller_spec.rb

```diff
@@ -3,225 +3,255 @@
 require 'spec_helper'
 
 describe Spree::CreditCardsController, type: :controller do
-  let(:user) { create(:user) }
-  let(:token) { "tok_234bd2c22" }
-
-  before do
-    Stripe.api_key = "sk_test_12345"
-    allow(controller).to receive(:spree_current_user) { user }
-  end
-
-  describe "#new_from_token" do
-    let(:params) do
-      {
-        format: :json,
-        exp_month: 12,
-        exp_year: Time.now.year.next,
-        last4: 4242,
-        token: token,
-        cc_type: "visa"
-      }
-    end
+  describe "using VCR", :vcr do
```

Comment:
The diff shown by github looks quite messy... But what actually is done here is separating the spec into:
1) `describe "using VCR", :vcr do` (here)